core/local/atom_watcher.js

```diff
@@ -80,6 +80,7 @@ module.exports = class AtomWatcher {
   events: EventEmitter
   checksumer: Checksumer
   producer: Producer
+  state: Object
```

Comment:
`state` looks good from the steps point of view, but `stepsState` would look better from the watcher point of view...

Reply:
Having a different name complexifies the different function calls. So either we use `stepsState` everywhere or we stick to `state`.
I'd rather do the later but I guess it's a matter of taste.